concepts/policies/README.md

```diff
@@ -5,16 +5,115 @@
 ## What is a Policy
 
 ### Introduction
-Policy are <strong>rules</strong>, that  yo can apply on <strong>Controllers</strong> and <strong>Functions</strong>.
-Policies provide <strong>access controll</strong>, <strong>autorization</strong> or any <strong>processing</strong> that can be done before accessing controller function.
+A Policy is <strong>function</strong>, that executed before accessing <strong>Controllers</strong>.
+With Policies you can setup <strong>access controll</strong>, <strong>autorization</strong> or any <strong>processing</strong> that can be done before accessing controller function.
 
-You can use Policies for whatever you want, the only convention is that policies hat to been RULES.
-
-In some case when you get complex treatment the best practice is to get short lines and undertandable code in policy, the checks and bool functions or processing into Services section of your app.
+You can use Policies for whatever you want, there is no convention.
 
 ### Examples
+
+#### Authorization Example
+
 For a policy that named isAdmin, we could write a function that allow access if user is admin and return a 403 if user is not an admin.
 
+#### Auth & Processing Example
+
 An other example can be a processing before access the controller, like refreshTokens, that automaticaly refresh authentication tokens for a user, if the user has not disconect from app, have same ip and has an outdated token that is valid, the result can be let accessing controllers or return a 403 if the user has disconected.    
 
 ## How to use it in Trails
+
+### Fastest Way : Use Yeoman
+
+```Bash
+  yo trails:policy ExamplePolicy
+```
+
+With Yeoman you don't need to create the file and include it in index.js
+
+### First Step : Write your policy
+
+The convention is that policies has to be written in api/policies.
+
+#### 1 - Using Hapi
+
+```JavaScript
+  'use strict'
+  const _ = require('lodash')
```

Comment:
Ho I didn't see it ^^ no need to have this just put `secret != ''` it's enough, user is here to learn Trails policy just keep it simple (in case he doesn't know about lodash...)